src/Sulu/Bundle/AutomationBundle/Serializer/TaskSerializerSubscriber.php

```diff
@@ -63,9 +82,14 @@ public function onTaskSerialize(ObjectEvent $event)
         }
 
         $handler = $this->handlerFactory->create($object->getHandlerClass());
-
         if ($handler instanceof AutomationTaskHandlerInterface) {
             $event->getVisitor()->addData('taskName', $handler->getConfiguration()->getTitle());
         }
+
+        $task = $this->taskRepository->findByUuid($object->getTaskId());
+        $executions = $this->taskExecutionRepository->findByTask($task);
```

Comment:
Can't this be done in a more performant way? This will result in quite a lot database requests. What about introducing a `TaskCollection` or something similar, which will do one query to retrieve all the required tasks? Resp. why do you need to reload the task object, shouldn't it be the one you retrieve by `$event->getObject()`?